src/next/azext_next/custom.py

```diff
@@ -2,60 +2,103 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-import re
 import json
+import re
 
+from azure.cli.core import telemetry
 from azure.cli.core.azclierror import RecommendationError
+from azure.cli.core.style import Style, print_styled_text
+from colorama import Fore, init
```

Comment:
Could we drop the usage of `colorama` as `Scenario Guide` does?

Reply:
> Could we drop the usage of `colorama` as `Scenario Guide` does?

Done